config/default_with_tools.yaml

```diff
@@ -29,6 +28,8 @@ instance_template: |-
   3. CWD is the directory of the repo you are supposed to edit. Only modify files inside this directory. Always provide absolute file paths, prefixed with $PWD.
   4. When editing files, it is easy to accidentally specify a wrong line number or to write code with incorrect indentation. Always check the code after you issue an edit to make sure that it reflects what you wanted to accomplish. If it didn't, issue another command to fix it.
 
+  # HOW TO BEGIN
+  To start things off, run the 'tdd_repro' command to reproduce the issue.
```

Comment:
1. Can we move this to `_tdd_repro_prompt.md`?
2. Does this always work? Generally speaking, not all instructions in a convoluted prompt will be followed, which is why I hardcoded `_make_initial_tdd_result`.

Reply:
re: moving it, I'm not sure - I wanted it to be at the very bottom of the prompt so it's close to the last thing the model receives.  Was hoping that would guarantee that it always works (it definitely has in my limited testing).  I'll try it in the moved location.

Reply:
> Generally speaking, not all instructions in a convoluted prompt will be followed

I think in this case we should endeavor to make sure ours will be (by changing the prompt further if we need to.)

Reply:
I'm all for it. Don't know how to (yet???), but looking forward to a more reliable AI buddy!